projects/hslayers/src/components/save-map/layman.service.ts

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable prettier/prettier */
```

Comment:
what was wrong with prettier in this file? I prefer to disable single lines if the rules are going to be messy somehow.

Reply:
Once again a change I havent even realized I made. Must have missclicked 'Fix all prettier..' . Will get rid of that, thanks